app/models/annotation/nml/NmlWriter.scala

```diff
@@ -22,6 +22,7 @@ case class NmlParameters(
     dataSetName: String,
     organizationName: String,
     description: Option[String],
+    wkUrl: String,
```

Comment:
I would probably have injected the config into the NmlWriter rather than passing this as a parameter. But I assume you did it this way for easier testing? Fine by me then.